SPECS/protobuf/protobuf.spec

```
@@ -108,6 +109,9 @@ popd
 %{python3_sitelib}/*
 
 %changelog
+* Mon Oct 09 2023 Mykhailo Bykhovtsev <mbykhovtsev@microsft.com> - 3.18.3-1
```

```suggestion
* Mon Oct 09 2023 Mykhailo Bykhovtsev <mbykhovtsev@microsft.com> - 3.17.3-3
```

Please also update the `Release` tag.